src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -854,5 +860,14 @@ public FormValidation doCheckWindowsPrivateKeyCredentialsId(@AncestorInPath Jenk
             }
             return FormValidation.ok();
         }
+
+        public FormValidation doCheckCreateSnapshot(@AncestorInPath Jenkins context,
+                                                                             @QueryParameter boolean value,
+                                                                             @QueryParameter("oneShot") boolean oneShot) {
+            if (!oneShot && value) {
+                return FormValidation.error("One-shot must be enabled to create snapshots");
```

Comment:
This error message needs to live in Message.properties for i18ln.

Reply:
Moved